website/docs/sql-function-glossary/sql-comments.md

```diff
@@ -0,0 +1,58 @@
+---
+id: sql-comments
+title: SQL Comments
+description: A CTE is a temporary result set that can be used in a SQL query. You can think of a CTE as a separate, smaller query within the larger query you’re building up.
+---
+
+<head>
+    <title>Working with SQL comments</title>
+</head>
+
+SQL comments…a two-folded thing: Are we talking about comments inline in SQL? Or comments on a table or view in the database?
+
+Why not both!
+
+In this page, we’ll unpack how to create both inline and database object-level comments, general best practices around SQL comments, and how dbt can help you improve (and version-control) your comments.
+
+## How to create SQL comments
+
+Inline SQL comments will begin with two dashes (--) in front of them in a query or dbt model; any text following these dashes is therefore what you’d call “commented out.” For longer, multi-line comments, you’ll typically see this syntax `/* your multi-line comment here */` used.
+
+### SQL comment example
+
+```sql
+/* these lines form a multi-line SQL comment; if it’s uncommented, 
+it will make this query error out */
+select
+	customer_id,
+	-- order_id, this row is commented out
+	order_date
+from {{ ref ('orders') }}
+```
+
+In practice, you’ll likely see SQL comments at the beginning of complex code logic, to help future developers or even advanced business users understand what specific blocks of code are accomplishing. Other times, you’ll see comments like the code above, that are commenting out lines no longer needed (or in existence) for that query or model. We’ll dive more into best practices around inline comments later on this page.
+
```

Comment:
May want to put these two examples under separate H3s for clarity, as they're very diff use cases

Reply:
Believe this is still open, if we want to break out the page in this way